### PR TITLE
Nolonely-Add Lottie Animation & Responsive Images 

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
1. Replace GIF with Lottie Animation.
2. Make images responsive according to resolution- 400w, 800w,1400w,2400w.
3. Remove Unused code.

**Lighthouse performance:**

Mobile: 74-80
Desktop:99

### CODE REVIEW

File: vue-frontend/src/components/portfolio/DesignSection.vue [hunk 1]
```diff
@@ -26,23 +26,21 @@
...
+    calc(100vw - 32px)"
         class="tw-w-full tw-h-full tw-object-cover"
         :alt="response.alt"
       />
     </aspect-ratio>
-    <img
-      v-if="response[0].gif"
-      :src="response[0].gif"
-      class="tw-absolute tw-inset-0 tw-left-auto tw-h-2/4 tw-w-2/6 tw-object-cover cycling-animation"
+    <LottieAnimation
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/DesignSection.vue:39]: Have you checked on real mobile devices? I think they not working properly on mobiles

File: vue-frontend/src/nolonely-data.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import landingBackground400w from "@/assets/images/portfolio/nolonely/landing/background-400w.webp";
```
Comment (cp-sumi-k) [inline vue-frontend/src/nolonely-data.js:1]: I think you have deleted 400 and 800 width images. Please check all images once

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/portfolio/DesignSection.vue (hunk 1 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/portfolio/DesignSection.vue:39]: its working well in all devices even in IOS also.

File: vue-frontend/src/nolonely-data.js (hunk 2 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/nolonely-data.js:1]: All the files are there.I have checked thoroughly.

File: vue-frontend/src/components/portfolio/LandingSection.vue [hunk 3]
```diff
@@ -14,14 +14,8 @@
...
-        :srcset="`${response.backgroundImage[0]} 400w, ${response.backgroundImage[1]} 800w, ${response.backgroundImage[2]} 1200w, ${response.backgroundImage[3]} 1600w`"
-        sizes="(min-width: 992px) 45vw, 100vw"
-        class="tw-w-full tw-h-full tw-object-cover"
-        :alt="response.alt"
-      />
-      <img
-        v-else
-        :src="response.backgroundImage"
+        :srcset="`${response.backgroundImage[0]} 400w, ${response.backgroundImage[1]} 800w, ${response.backgroundImage[2]} 1400w, ${response.backgroundImage[3]} 2400w`"
+        sizes="2200px,720px,520px"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/LandingSection.vue:18]: I think without media queries, sizes won't work. Shouldn't it?

Comment (cp-priyanka-g) [reply vue-frontend/src/components/portfolio/LandingSection.vue:18]: yes

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**